test/metabase/test/util.clj

```diff
@@ -389,13 +413,27 @@
   To temporarily override the value of *read-only* env vars, use [[with-temp-env-var-value]]."
   [[setting-k value & more :as bindings] & body]
   (assert (even? (count bindings)) "mismatched setting/value pairs: is each setting name followed by a value?")
+  (test-runner.parallel/assert-test-is-not-parallel "with-temporary-setting-vales")
   (if (empty? bindings)
     `(do ~@body)
     `(do-with-temporary-setting-value ~(keyword setting-k) ~value
        (fn []
          (with-temporary-setting-values ~more
            ~@body)))))
 
+(defmacro with-temporary-raw-setting-values
+  "Like `with-temporary-raw-setting-values` but works with raw value and it allows undefined settings."
```

Comment:
```suggestion
  "Like `with-temporary-setting-values` but works with raw value and it allows undefined settings."
```

Reply:
maybe clarify what `undefined settings` means? i.e. "it can be used with setting names that are not defined with `defsetting`"

at first I read it to mean settings that have "undefined" values